src/bindings/python/tests/test_utils/test_utils.py

```diff
@@ -17,14 +17,100 @@
 from openvino.utils import deprecated
 
 
-def test_compare_models():
-    try:
-        from openvino.test_utils import compare_models
-        model = get_relu_model()
-        status, _ = compare_models(model, model)
-        assert status
-    except RuntimeError:
-        print("openvino.test_utils.compare_models is not available")  # noqa: T201
+def _compare_models(model_one: Model, model_two: Model, compare_names: bool = True) -> Tuple[bool, str]:  # noqa: C901 the function is too complex
```

Comment:
@itikhono @jane-intel please, take a look

Reply:
@itikhono @jane-intel please let me know directly if you have any concerns.